R/bm-dataset-taxi-2013.R

```diff
@@ -0,0 +1,106 @@
+#' Benchmark Taxi 2013 dataset reading
+#'
+#' @section Parameters:
+#' * `query` Name of a known query to run; see `dataset_taxi_2013$cases`
+#'
+#' @export
+dataset_taxi_2013 <- Benchmark(
+  "dataset_taxi_2013",
+  setup = function(query = names(dataset_taxi_2013$cases)) {
+    library("dplyr", warn.conflicts = FALSE)
+    dataset <- ensure_dataset("taxi_2013")
+    query <- dataset_taxi_2013$cases[[match.arg(query)]]
+
+    BenchEnvironment(
+      query = query,
+      dataset = dataset
+    )
+  },
+  before_each = {
+    result <- NULL
+  },
+  run = {
+    result <- query$query(dataset)
+  },
+  after_each = {
+    query$assert(result)
+  },
+  cases = list(
+    vignette = list(
```

Comment:
I wonder if this should be changed from "vignette" as that refers specifically to the vignette query.

Reply:
Maybe "aggregation"?

Reply:
yeah maybe i should just give up on trying to keep the names here parallel to taxi_parquet